changefeedtrigger/changefeedSample.py

```diff
@@ -0,0 +1,91 @@
+from .settings import *
+from azure.storage.blob.changefeed import ChangeFeedClient
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+import logging
+
+
+def main(mytimer: func.TimerRequest) -> None:
+
+    # create blob service client to access connection string
+    blob_service_client = BlobServiceClient.from_connection_string(conn_str=CONNECTION_STRING)
+    change_feed_client = ChangeFeedClient.from_connection_string(conn_str=CONNECTION_STRING)
+
+    # create or get container to store cursor
+    container_name = "cursorstoragecontainer"
+    try:
+        token_container_client = blob_service_client.create_container(container_name)
+    except:
+        token_container_client = blob_service_client.get_container_client(container_name)
+
+    # call to method
+    cursor = get_cursor(token_container_client)
+    events = get_events(cursor, token_container_client, change_feed_client)
+    events = filter_events(events) # if filtering events is unwanted this method can be commented out
+ 
+    # log filtered event output
+    if len(events) == 0:
+        logging.info("\nNo events matching filters on current page...")
+    else:
+        logging.info("\nRecents event(s) matching filters on current page:")
+        count = 1
+        for e in events:
+            logging.info("%d: %s\n" % (count, e))
+            count += 1
+
+
+def get_cursor(token_client):
+    # this method is to grab and return the cursor
+    
+    # access blobs in container
+    blob_list = token_client.list_blobs()
+    tokens = []
+    for blob in blob_list:
+        tokens.append(blob.name)
+
+    return tokens
```

Comment:
it this function for detecting if 'cursorBlob' is in the blob list?

Reply:
Talked to Carmen offline, and we agreed to make it into try: except block

---
In reply to: [469609624](https://github.com/wastore/azure-storage-samples-for-python/pull/7#discussion_r469609624) [](ancestors = 469609624)